README.md

```diff
@@ -83,22 +83,19 @@ Include the dependency in your app `build.gradle`:
 
 ```groovy
 dependencies {
-    implementation 'com.schibstedspain.android:leku:7.0.1'
+    implementation 'com.schibstedspain.android:leku:7.1.0'
 }
 ```
 
 Alternatively, if you are using a different version of Google Play Services and AndroidX use this instead:
 
 ```groovy
-implementation ('com.schibstedspain.android:leku:7.0.1') {
+implementation ('com.schibstedspain.android:leku:7.1.0') {
```

Comment:
I'm going to release a new version that includes the removal of RxJava2 and Jetifier